patroni/api.py

```diff
@@ -592,6 +592,18 @@ def do_POST_failover(self, action='failover'):
     def do_POST_switchover(self):
         self.do_POST_failover(action='switchover')
 
+    @check_access
```

Comment:
TODO: `restapi.allowlist_include_members` probably should take into account members from all Citus groups.